src/core/ui.js

```diff
@@ -96,7 +96,9 @@ function trapFocus(element) {
   );
   const firstFocusableEl = focusableEls[0];
   const lastFocusableEl = focusableEls[focusableEls.length - 1];
-
+  if (focusableEls) {
```

Comment:
`focusableEls` is always truthy
```suggestion
  if (firstFocusableEl) {
```